README.md

```diff
@@ -3,7 +3,8 @@
 [![rx-angular](https://circleci.com/gh/BioPhoton/rx-angular.svg?style=shield)](https://circleci.com/gh/BioPhoton/rx-angular)
 
 
-This repository holds a set of helpers to create fully reactive applications.
+This repository holds a set of helpers to create **fully reactive** as wel as  **fully zone-less** applications.
```

Comment:
```suggestion
This repository holds a set of helpers to create **fully reactive** as well as  **fully zone-less** applications.
```

didn't we talk about those bold stuff? :D

Reply:
Ture :D lets fix it in a release branch.